internal/repository/employee/employee_test.go

```diff
@@ -0,0 +1,439 @@
+package employee
```

Comment:
although it's "kinda ok", but I would suggest utilizing your knowledge of build constraints here, since it's an integration test

Reply:
fixed